packages/@rnw-scripts/doxysaurus/src/doxygen.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * This file contains some code taken from the doxygen project.
+ * Copyright for the doxygen package code:
+ * Copyright (c) 2016 Ivan Maria <iv.ivan.maria@gmail.com>;
+ *
+ * @format
+ */
+
+//
+// Generates Doxygen documentation XML files.
+//
+
+// @ts-ignore (no typings for doxygen)
+import constants = require('doxygen/lib/constants');
```

Comment:
`import x = require(y)` is a special syntax for CommonJS interop combied with typings. Not widely supported (e.g. Babel will complain if it sees it).

Replace with `import * as constants from 'doxygen/lib/constants'`. Will do the same thing.

Reply:
Good point: fixed